app/views/users/edit.html.erb

```diff
@@ -1,40 +1,7 @@
 <h1><%= I18n.t('user.edit.edit_profile_heading') %></h1>
```

Comment:
This would not be the `Edit profile page` then, would it?

Reply:
Resolved via e2b32193a7ae94f44a2f5f7438306fe9e84fcd19